services/ui-src/src/components/export/ExportedEntityDetailsTableRow.tsx

```diff
@@ -41,15 +41,15 @@ export const ExportedEntityDetailsTableRow = ({
       {/* label column/cell */}
       <Td sx={sx.labelColumn}>
         {formFieldInfo.label || formFieldInfo.hint ? (
-          <Text>
+          <Box>
```

Comment:
Fixes `Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.` and `Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.`